python/tests/res/enkf/test_enkf.py

```diff
@@ -90,7 +90,7 @@ def test_invalid_res_config(self):
         with TestAreaContext("enkf_test") as work_area:
             with self.assertRaises(TypeError):
                 work_area.copy_directory(self.case_directory)
-                main = EnKFMain("simple_config/minimum_config", res_config=self)
+                main = EnKFMain(res_config=self)
```

Comment:
I just wanted to provoke a `TypeError` by not giving a ResConfig and had the test instance itself at hand.. I can change it if you like me to.